### PR TITLE
Added and updating patches

### LINKED ISSUE #429: MGS 3 Subsistence: Disable ws for problem?
https://github.com/PCSX2/pcsx2_patches/pull/428
Where has the error been seen in MGS3 subsistence? I finished it from start to finish a few months ago, and I was correcting it on the fly without any problem... and the ShizCalev himself confirms it.
If any anomaly has been detected, where is it located? In order to correct it, I would fix it in a moment.

### PR DESCRIPTION
**Pull request by PeterDelta**
I have added tested patches and disabled some widescreens because they cause problems or are incomplete, I have not been able to fix them.

Relevant changes:
Driven: Disabled WS. Missing render fix
Hobbit: Fixed the problem with the strings, they no longer move.
The Simpsons, the Game: The current fps patch consumes a lot of resources and slows down the game, fixed

Arthur and the Minimoys: Image scrolls up
Arthur and the Minimoys_SLES-54420_20240819014238
Current patch

Arthur and the Minimoys_SLES-54420_20240828110022
4:3 original

Arthur and the Minimoys_SLES-54420_20240929205400
Update patch
____________________________

Monster House: Disabled WS. After the first level, the game breaks on the other levels
Monster House_SLES-54215_20240929175629
____________________________
Shadow the Hedgehog: The region code in the repository is incorrect, corrected and adjusted the values ​​to match the perspective
Captura de pantalla 2024-10-01 145901
____________________________
Updates MGS3 Snake Eater PAL-S/I and MGS3 Subsistence PAL-S and  NTSC-U
Fixes #429 

I've completed the game again from start to finish and have fixed a couple of very subtle scenes that I missed, all the reflections are as they should be. I've also updated the fps patch, at certain times it would return to 25/30, now the whole game runs at 50/60 fps.

Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241015103934
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241015104135
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241015105558
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241015105608
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241015110739
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241015110935
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241017151242
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241017151445
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241017151810
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241017162514
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241017162635
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241017224231
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241017224255
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241017224304
Metal Gear Solid 3 - Subsistence  Disc 1 of 3 _SLUS-21359_20241017225732

### CODE REVIEW

File: patches/SLES-50131_FEA030CB.pnach [hunk 1]
```diff
@@ -20,4 +20,12 @@ patch=1,EE,0039ed30,word,080c8b62 // 00000000
 //patch=1,EE,0039ed2c,word,27b001e0 // 00000000
 //patch=1,EE,0039ed30,word,080c8b62 // 00000000
```
Comment (pgert) [inline patches/SLES-50131_FEA030CB.pnach:22]: [Widescreen 16:10]
gsaspectratio=Stretch
author=ElHecht
description=Renders the game in 16:10 aspect ratio.
patch=1,EE,00322d84,word,080e7b49
patch=1,EE,00322d88,word,00000000
patch=1,EE,0039ed24,word,3c033f99
patch=1,EE,0039ed28,word,3463999a
patch=1,EE,0039ed2c,word,27b001e0
patch=1,EE,0039ed30,word,080c8b62

[Widescreen 15:10]
gsaspectratio=Stretch
author=ElHecht
// converted by pgert
description=Renders the game in 15:10 aspect ratio.
patch=1,EE,00322d84,word,080e7b49
patch=1,EE,00322d88,word,00000000
patch=1,EE,0039ed24,word,3c033F90
patch=1,EE,0039ed28,word,34630000
patch=1,EE,0039ed2c,word,27b001e0
patch=1,EE,0039ed30,word,080c8b62

File: patches/SLES-53242_7B94480C.pnach [hunk 2]
```diff
@@ -1,10 +1,9 @@
-gametitle=Shadow the Hedgehog SLES_532.42
+gametitle=Shadow the Hedgehog (PAL-M) SLES-53542 7B94480C
```
Comment (pgert) [inline patches/SLES-53242_7B94480C.pnach:2]: Shadow the Hedgehog * PAL-? * SLES-53242 * 7B94480C
 -> DreamWorks Madagascar * PAL-P * SLES-53242 * 7B94480C
According to GameIndex and redump: http://redump.org/disc/67995/

File: patches/SLES-53695_CED08066.pnach [hunk 3]
```diff
@@ -8,4 +8,7 @@ author=El_Patas
 patch=1,EE,003E3C20,word,3C013FE3 //3C013FAA
 patch=1,EE,003E3C24,word,3421388E //3421AAAB
```
Comment (pgert) [inline patches/SLES-53695_CED08066.pnach:10]: [Widescreen 16:9]
gsaspectratio=16:9
author=El_Patas
// corrected by pgert
patch=1,EE,003E3C20,word,3C013FE3 // 3C013FAA
patch=1,EE,003E3C24,word,34218E39 // 3421AAAB

[Widescreen 16:10]
gsaspectratio=Stretch
author=El_Patas
// converted by pgert
patch=1,EE,003E3C20,word,3C013FAC
patch=1,EE,003E3C24,word,3421CCCD

[Widescreen 15:10]
gsaspectratio=Stretch
author=El_Patas
// converted by pgert
patch=1,EE,003E3C20,word,3C013FA2
patch=1,EE,003E3C24,word,34210000

**Review by JordanTheToaster — APPROVED** [review]

**Comment by ShizCalev** [comment]
`I've also updated the fps patch, at certain times it would return to 25/30, now the whole game runs at 50/60 fps.`
Just a heads up, there's a text animation that runs at double speed due to the 60 FPS patch during the intro cutscene. 
#11805

**Pull request merged**